arm-apimanagement/2016-10-10/swagger/apimdeployment.json

```diff
@@ -0,0 +1,1270 @@
+          "default": 1
+        }
+      },
+      "required": [
+        "name"
+      ],
+      "description": "API Management service resource SKU properties."
+    },
+    "ApiManagementServiceResource": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The ID of the created API Management service.",
+          "readOnly": true
+        },
+        "location": {
+          "type": "string",
+          "description": "Datacenter location of the API Management service."
+        },
+        "name": {
+          "type": "string",
+          "description": "Name of the API Management service.",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "Resource type of the API Management service.",
+          "readOnly": true
+        },
+        "etag": {
+          "type": "string",
+          "description": "ETag of the resource."
+        }
+      },
+      "allOf": [
+          {
+            "$ref" : "#/definitions/ApiManagementServiceBaseParameters"
+          }
+       ],
+      "required": [
+        "sku",
+        "properties",        
+        "location"
+      ],
+      "description": "Description of an API Management service resource."
+    },
+    "ApiManagementServiceBaseParameters": {
+      "properties": {        
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "API Management service tags. A maximum of 10 tags can be provided for a resource, and each tag must have a key no greater than 128 characters (and a value no greater than 256 characters)."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ApiManagementServiceProperties",
+          "description": "Properties of the API Management service."
+        },
+        "sku": {
+          "$ref": "#/definitions/ApiManagementServiceSkuProperties",
+          "description": "SKU properties of the API Management service."
+        }
+      },
+      "description": "Parameters supplied to the Update API Management service operation."
+    },
+    "ApiManagementServiceListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApiManagementServiceResource"
+          },
+          "description": "Result of the List API Management services operation."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to the next set of results. Not empty if Value contains incomplete list of API Management services."
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "description": "The response of the List API Management services operation."
+    },
+    "ApiManagementServiceGetSsoTokenResult": {
+      "properties": {
+        "redirect_uri": {
```

Comment:
We will take this as feedback and update it in next version of the API. This api-version is already being used, we are just updating the swagger for documentation purpose.

---
In reply to: [106764472](https://github.com/Azure/azure-rest-api-specs/pull/1039#discussion_r106764472) [](ancestors = 106764472)